server.py

```diff
@@ -1,58 +1,78 @@
-import threading
-from flask import Flask
-from flask_socketio import SocketIO, emit
-from canvas_board import CanvasBoard, CanvasBoardEncoder
+import random, string
+from flask import Flask, jsonify
+from flask_socketio import SocketIO, emit, join_room, leave_room
+from canvas_board import CanvasBoard
+from json import dumps
 
 # constants
 PORT = 8080
 HEIGHT = 500
 WIDTH = 500
+ID_LENGTH = 7
 
 # create Flask object
-app = Flask(__name__)
+app = Flask(__name__, template_folder='static/', static_folder='static/')

```

Comment:
Need to delete extra arguments 'template_folder' and 'static_folder'

Reply:
We don't need `template_folder` since we are not using templates and we don't need `static_folder` since it defaults to "static".

Also, since the frontend is hardcoded to use the address of the hosted server, you will need to change it to localhost when testing locally.  I will look into fixing this later though.